test/Yoti.Auth.Tests/YotiAttributeValue_Tests.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using static Yoti.Auth.YotiAttributeValue;
+
+namespace Yoti.Auth.Tests
+{
+    [TestClass]
+    public class YotiAttributeValue_Tests
+    {
+        [TestMethod]
+        public void YotiAttributeValue_JpegData_DoesntThrowException()
+        {
+            string jpegData = "jpegData";
+
+            YotiAttributeValue yotiAttributeValue = new YotiAttributeValue(
+                TypeEnum.Jpeg,
+                jpegData);
+        }
+
+        [TestMethod]
+        public void YotiAttributeValue_PngData_DoesntThrowException()
```

Comment:
I realise the elements that this test covers, as well as the test below, are covered by the most recent commit, so we can remove them.